.changeset/blue-stingrays-sing.md

```diff
@@ -0,0 +1,5 @@
+---
+"@getflip/swirl-components": minor
+---
+
```

Comment:
I have merged the branches into the dialog branch, because I needed these components. As soon as the button and group are merged into main, these changes will disappear from this PR.